src/views/open-api-docs-view-v2/index.tsx

```diff
@@ -19,40 +23,32 @@ import type { OpenApiDocsViewV2Props } from './types'
  */
 export default function OpenApiDocsViewV2({
```

Comment:
In the main view here, we're now consuming the new props passed from `getStaticProps`, and rendering those with the newly established components.